// _@vuepress_core@1.7.1@@vuepress/core/.temp/internal/routes.js
/**
 * Generated by "@vuepress/internal-routes"
 */

import { injectComponentOption, ensureAsyncComponentsLoaded } from '@app/util'
import rootMixins from '@internal/root-mixins'
import GlobalLayout from "/Users/mac/workspace/github/wheel/node_modules/_@vuepress_core@1.7.1@@vuepress/core/lib/client/components/GlobalLayout.vue"

injectComponentOption(GlobalLayout, 'mixins', rootMixins)
export const routes = [
  {
    name: "v-6496f926",
    path: "/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-6496f926").then(next)
    },
  },
  {
    path: "/index.html",
    redirect: "/"
  },
  {
    name: "v-d5415866",
    path: "/plugin/jenkins/extension.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-d5415866").then(next)
    },
  },
  {
    name: "v-24751049",
    path: "/plugin/obfuscator/crypto.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-24751049").then(next)
    },
  },
  {
    name: "v-39e43dd6",
    path: "/vueRouter/extension.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-39e43dd6").then(next)
    },
  },
  {
    name: "v-2ac596b5",
    path: "/plugin/obfuscator/extension.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-2ac596b5").then(next)
    },
  },
  {
    path: '*',
    component: GlobalLayout
  }
]
